src/Utilities/Compiler/WellKnownTypeNames.cs

```diff
@@ -5,6 +5,7 @@ namespace Analyzer.Utilities
     internal static class WellKnownTypeNames
     {
         public const string SystemAppContext = "System.AppContext";
+        public const string SystemBuffersMemoryManager1 = "System.Buffers.MemoryManager`1";
```

Comment:
Is the guidance to always add these type constants for any type an analyzer needs to check for, or is this list for commonly inspected types?

Reply:
I didn't see any guidance about that, seems all types used are in here no matter common or not